clinica/utils/input_files.py

```diff
@@ -0,0 +1,128 @@
+# coding: utf8
+
+"""
+Describe files to grab, to use with inputs.clinica_file_reader() and inputs.clinica_group_reader()
+
+"""
+
+""" T1w """
+
+# BIDS
+
+T1W_NII = {'pattern': 'sub-*_ses-*_t1w.nii*',
+           'description': 'T1w MRI'}
+
+# FreeSurfer
+
+T1_FS_WM = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/mri/wm.seg.mgz',
+            'description': 'segmentation of white matter (mri/wm.seg.mgz)',
+            'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_BRAIN = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/mri/brain.mgz',
+               'description': ' extracted brain from T1w MRI',
+               'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_ORIG_NU = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/mri/orig_nu.mgz',
+                 'description': 'intensity normalized volume generated after correction for'
+                                ' non-uniformity in FreeSurfer (file mri/orig_nu.mgz) ',
+                 'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_WM_SURF_R = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/surf/rh.white',
+                   'description': 'right white matter/gray matter border surface.',
+                   'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_WM_SURF_L = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/surf/lh.white',
+                   'description': 'left white matter/gray matter border surface.',
+                   'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_DESTRIEUX = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/mri/aparc.a2009s+aseg.mgz',
+                   'description': 'Destrieux-based segmentation (mri/aparc.a2009s+aseg.mgz)',
+                   'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_DESTRIEUX_SURF_L = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/label/lh.aparc.a2009s.annot',
```

Comment:
These are discrete parcellations and not surfaces. From [BEP 11](https://docs.google.com/document/d/1YG2g4UkEio4t_STIBOqYOwneLEs1emHIXbGKynx7V0Y/edit#heading=h.y9xsm9g2uzp5), you can find use `DPARC` for discrete parcellation or simply `PARC`, it is up to you:
```suggestion
T1_FS_DESTRIEUX_PARC_L = {'pattern': 't1/freesurfer_cross_sectional/sub-*_ses-*/label/lh.aparc.a2009s.annot',
```
(Final update for Desikan needs to be done and we can merge this PR)

Reply:
Done